test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -258,13 +277,34 @@ describe('33acrossBidAdapter:', function () {
       }
     ];
 
+    this.addBid = (bidParams = {}) => {
+      bidRequests.push({
+        bidId: 'b2',
```

Comment:
Optional:

should we use a counter or Date.getTime to generate a unique Id ? probably makes the unit tests unnecessarily more complex but at the same time we could use this `addBid` function to add the default bid that's always set when the object is instantiated.

Reply:
We could, except we will have to explicitly exclude `bidId1 in our assertions why comparing expected result against actual otherwise test might fail.